spec.html

```diff
@@ -40118,16 +40120,26 @@ <h1>Promise.all ( _iterable_ )</h1>
           <p>The `all` function requires its *this* value to be a constructor function that supports the parameter conventions of the Promise constructor.</p>
         </emu-note>
 
+        <emu-clause id="sec-getpromiseresolve" aoid="GetPromiseResolve">
+          <h1>Runtime Semantics: GetPromiseResolve ( _promiseConstructor_ )</h1>
+          <p>The abstract operation GetPromiseResolve takes argument _promiseConstructor_. It performs the following steps when called:</p>
+          <emu-alg>
+            1. Assert: IsConstructor(_promiseConstructor_) is *true*.
+            1. Let _promiseResolve_ be ? Get(_promiseConstructor_, *"resolve"*).
+            1. If IsCallable(_promiseResolve_) is *false*, throw a *TypeError* exception.
+            1. Return _promiseResolve_.
+          </emu-alg>
+        </emu-clause>
+
         <emu-clause id="sec-performpromiseall" aoid="PerformPromiseAll">
-          <h1>Runtime Semantics: PerformPromiseAll ( _iteratorRecord_, _constructor_, _resultCapability_ )</h1>
-          <p>The abstract operation PerformPromiseAll takes arguments _iteratorRecord_, _constructor_, and _resultCapability_. It performs the following steps when called:</p>
+          <h1>Runtime Semantics: PerformPromiseAll ( _iteratorRecord_, _constructor_, _resultCapability_, _promiseResolve_ )</h1>
+          <p>The abstract operation PerformPromiseAll takes arguments _iteratorRecord_, _constructor_, _resultCapability_, and _promiseResolve. It performs the following steps when called:</p>
```

Comment:
```suggestion
          <p>The abstract operation PerformPromiseAll takes arguments _iteratorRecord_, _constructor_, _resultCapability_, and _promiseResolve_. It performs the following steps when called:</p>
```

(The linter doesn't yet know how to catch these, sorry. Someday!)